convert2rhel/pkghandler.py

```diff
@@ -706,8 +706,9 @@ def _get_package_repositories(pkgs):
             repositories_mapping[package] = "N/A"
     else:
         for line in output:
-            if "C2R" in line:
-                split_output = line.lstrip("C2R ").split("&")
+            line = line.lstrip()
```

Comment:
I don't think we'll encounter this in the wild tbh, it's just something within our own tests, but doesn't hurt so let's leave it be for now